.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Recipes added with this pull request (max 5):
+ 
+ - package1
+ - package2
```

Comment:
It helps to see which packages are being added without having to go through the "files changed" tab.
P.S. I pushed a new commit taking your comments in consideration.